src/components/10-atoms/progress-bar/index.js

```diff
@@ -0,0 +1,82 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAProgressBar extends LitElement {
+  static get tagName() {
+    return 'axa-progress-bar';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean, defaultValue: false },
+      noBorderRadius: { type: Boolean },
+      value: { type: Number },
+      max: { type: Number },
+      text: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  calculatePercantage() {
+    if (this.max !== 0) {
+      if (this.value > this.max) {
+        return 100;
+      }
+      return (this.value / this.max) * 100;
+    } else if (this.value > 100) {
+      return 100;
+    }
+    return this.value;
+  }
+
+  render() {
+    const classesBorder = {
+      'a-progress-bar__small': this.small,
+      'a-progress-bar__no-border-radius': this.noBorderRadius,
+    };
+
+    const classesLoader = {
+      'a-progress-bar__small': this.small,
+      'a-progress-bar__no-border-radius-loader': this.noBorderRadius,
+      'a-progress-bar__no-border-radius-loader-max':
```

Comment:
a-progress-bar__loader--no-border-radius-max

I'm not sure about this name and also don't understand what "max" implies.

Reply:
max = max value from the bar (same attribute name in `<progress>`)

This class is needed because when the loading bar is at 100% it can't be round anymore.